src/pages/TCY/components/Claim/ClaimConfirm.tsx

```diff
@@ -142,16 +145,30 @@ export const ClaimConfirm = ({ claim, setClaimTxid }: ClaimConfirmProps) => {
       if (!txid) return
 
       setClaimTxid(txid)
-      navigate(TCYClaimRoute.Status)
+
+      dispatch(
+        actionSlice.actions.upsertAction({
+          id: claim.l1_address,
```

Comment:
note, `l1_address` is the id across the board since that the lowest common discriminator for a claim, which are unique by L1 address upstream